agent-tooling/src/main/java/io/opentelemetry/auto/tooling/ExporterClassLoader.java

```diff
@@ -48,7 +48,9 @@
           rule(
               "#io.opentelemetry.metrics",
               "#io.opentelemetry.auto.shaded.io.opentelemetry.metrics"),
-          rule("#io.opentelemetry.trace", "#io.opentelemetry.auto.shaded.io.opentelemetry.trace"));
+          rule("#io.opentelemetry.trace", "#io.opentelemetry.auto.shaded.io.opentelemetry.trace"),
+          rule("#java.util.logging.Logger", "#io.opentelemetry.auto.bootstrap.PatchLogger"),
+          rule("#org.slf4j", "#io.opentelemetry.auto.slf4j"));
```

Comment:
why not `io.opentelemetry.auto.shaded.org.slf4j`?

Reply:
it needs to match the current slf4j shading in the bootstrap class loader (though fair to ask why the current shading is like this)